### ISSUE TITLE
Fix false positive in rule ForbidReturnInFinalBlockCheck

### ISSUE THREAD

**Issue by Ravynne** [sevntu-checkstyle/sevntu.checkstyle#274]
The code below produces the error "Finally block should not contain return statements". I reduced the code down to the smallest code that would produce the error. The keyword break is significant here.

``` java
    private static void validate() {
        try {
            while (true) {
                break;
            }
        }
        finally {
        }
    }
```
